collated/test/A0124333U.md

```diff
@@ -0,0 +1,568 @@
+# A0124333U
+###### \java\guitests\EditCommandTest.java
+``` java
+ */
+
+import org.junit.Test;
+
+import tars.commons.exceptions.IllegalValueException;
+import tars.logic.commands.EditCommand;
+import tars.model.task.Name;
+import tars.model.task.Priority;
+import tars.testutil.TestTask;
+import tars.testutil.TestUtil;
```

Comment:
do note the java coding stnd applies to test codes too ~ @weikangchia @jaeoheeail @johnervan @origiri